Documentation/gettingstarted/ipvlan.rst

```diff
@@ -0,0 +1,116 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _ipvlan:
+
+*********************
+Using ipvlan datapath
+*********************
+
+This guide explains how to configure Cilium to set up an ipvlan-based
+datapath instead of the default veth-based one.
+
+.. note::
+
+    This is a beta feature. Please provide feedback and file a GitHub issue
+    if you experience any problems.
+
+First step is to download the Cilium Kubernetes descriptor:
+
+.. tabs::
+  .. group-tab:: K8s 1.13
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.13/cilium.yaml
+
+  .. group-tab:: K8s 1.12
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.12/cilium.yaml
+
+  .. group-tab:: K8s 1.11
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.11/cilium.yaml
+
+  .. group-tab:: K8s 1.10
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.10/cilium.yaml
+
+  .. group-tab:: K8s 1.9
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.9/cilium.yaml
+
+  .. group-tab:: K8s 1.8
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.8/cilium.yaml
+
+Edit the ``cilium-config`` ConfigMap in that file with the etcd server
+that is running in your cluster and set the option ``datapath-mode`` to
+``"ipvlan"``.
+
+It is also required to specify the ipvlan master device which typically
+points to a networking device that is facing the external network. This
+is done through setting ``ipvlan-master-device`` to the name of the
+networking device such as ``"eth0"`` or ``"bond0"``, for example. For
+deployment, this device must have consistent naming on each host.
```

Comment:
Replace this last sentence with something as: `Be aware this ConfigMap will be used by all nodes so it is required this device name is consistent on all nodes where you will deploy Cilium`